internal/host/host_test.go

```diff
@@ -607,6 +607,52 @@ var _ = Describe("reset host", func() {
 
 })
 
+var _ = Describe("register host", func() {
+	var (
+		ctx           = context.Background()
+		db            *gorm.DB
+		state         API
+		h             models.Host
+		eventsHandler events.Handler
+		dbName        = "reset_host"
```

Comment:
there is already a test that validate host registration, here you are using the same DB name, if those tests will run in parallel they will fail.
So please move the test into already existing tests or change the DB name to make it unique